### PR TITLE
Barrier: Use Boot AutoConfig for RabbitMQ

### PR DESCRIPTION
**Pull request by garyrussell**
Also support a browser GET.

### CODE REVIEW

File: basic/barrier/src/main/resources/META-INF/spring/integration/client-context.xml [hunk 1]
```diff
@@ -14,8 +14,14 @@
 	<int:channel id="requestChannel"/>
 
 	<int-http:outbound-gateway request-channel="requestChannel" 
-					url="http://localhost:8080/receiveGateway"
+					url="#{url}"
 					http-method="POST"
 					expected-response-type="java.lang.String"/>
 
+	<beans profile="default">
```
Comment (artembilan) [inline basic/barrier/src/main/resources/META-INF/spring/integration/client-context.xml]: I can merge it as is, but wonder, first of all, what you have had in mind introducing this `profile`-based configuration...

Comment (garyrussell) [inline basic/barrier/src/main/resources/META-INF/spring/integration/client-context.xml]: I was playing around with a `cloud` profile.

I decided not to pursue it in this PR but didn't see any reason not to show how different URLs can be used via profiles.

Comment (artembilan) [inline basic/barrier/src/main/resources/META-INF/spring/integration/client-context.xml]: Yes, that's fine. Merging as is after rebase....

**Comment by artembilan** [comment]
Merged as https://github.com/spring-projects/spring-integration-samples/commit/23f4a04661abf195a7eb0c7ed5744b844d916ef7

**Pull request closed without merging**